### PR TITLE
adding text about downloads

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

**Comment by antgonza** [comment]
BTW the new link looks like this:
new

File: qiita_pet/support_files/doc/source/faq.rst [hunk 1]
```diff
@@ -90,10 +90,26 @@ A few more instructions: for the example above the workflow should be:
...
-How to solve unzip errors?
---------------------------
-
-When downloading large zip files within Qiita there is a change that you will get
+How to solve download or unzip errors?
+--------------------------------------
+
+Dealing with large files might be daunting but, in general, following these
+instructions should make things easier. First, make sure that you have enough space
+for the zip download file; for the American Gut, you should have at least close to 700Gb
```
Comment (wasade) [inline qiita_pet/support_files/doc/source/faq.rst]: Would it be possible to, near the download link, just show the size of the zip file to download? The size estimate for the AGP will be incorrect pretty soon.

File: qiita_pet/support_files/doc/source/faq.rst [hunk 2]
```diff
@@ -90,10 +90,26 @@ A few more instructions: for the example above the workflow should be:
...
+but having close to a Tera (1000Gb) should be better as you will need to uncompress
+the zip file. Second, make sure that your computer has all the sleep settings turned off;
+for example, in a Mac, got to System Preferences, Energy Saver, Power Adapter and unselect
+the option of "Put hard disks to sleep when possible"; don't forget to save the settings.
+Third, download the file but point to the storage that you want to save your file in; using
+Chrome, right click on the download button and select "Save Link As ..."; and select the
+location where you have enough space (see point 1). Fourth, wait for the download to finish,
+around 13hrs in an OK USA Internet connection for the AGP but this will depend on your Internet
```
Comment (wasade) [inline qiita_pet/support_files/doc/source/faq.rst]: That's roughly 14MB/s which is a pretty nice connection, and many ISPs have total download limits per month. I'd omit this sentence... the browser should provide a download time estimate anyway?

File: qiita_pet/support_files/doc/source/faq.rst [hunk 3]
```diff
@@ -90,10 +90,26 @@ A few more instructions: for the example above the workflow should be:
...
+service. Finally, unzip de file with a newer version of zip.
```
Comment (wasade) [inline qiita_pet/support_files/doc/source/faq.rst]: de -> the

What is a newer version of zip?

Comment (wasade) [reply qiita_pet/support_files/doc/source/faq.rst]: Oh, I see below. Might want to mention "(see below)". Note that this is likely limited to versions of OSX. unzip 6.0.0 on 10.13 reports a compilation date of 2017, and the unzip 6.0.0 build on barnacle reports a compilation date of 2010

File: qiita_pet/support_files/doc/source/faq.rst (hunk 1 above)
Comment (antgonza) [reply qiita_pet/support_files/doc/source/faq.rst]: Just noticed that the full uncompressed + the zip is close to 2T; will change. BTW estimating the size of the zip is actually pretty hard to do accurately for the merged zip files but we could do ballpark estimates; let's discuss in the Wednesday qiita meeting. BTW there is an issue about this: https://github.com/biocore/qiita/issues/2133

File: qiita_pet/support_files/doc/source/faq.rst (hunk 2 above)
Comment (antgonza) [reply qiita_pet/support_files/doc/source/faq.rst]: Good point, will remove as arguably 14M is "basic" based https://www.speedtest.net/reports/ at least on the countries listed there ... but as always depends compared to what ...

File: qiita_pet/support_files/doc/source/faq.rst (hunk 3 above)
Comment (antgonza) [reply qiita_pet/support_files/doc/source/faq.rst]: ha! de! thanks for the info ...

File: qiita_pet/support_files/doc/source/faq.rst [hunk 4]
```diff
@@ -90,10 +90,26 @@ A few more instructions: for the example above the workflow should be:
...
-How to solve unzip errors?
---------------------------
-
-When downloading large zip files within Qiita there is a change that you will get
+How to solve download or unzip errors?
+--------------------------------------
+
+Dealing with large files might be daunting but, in general, following these
+instructions should make things easier. First, make sure that you have enough space
+for the zip download file; for the American Gut, you should have at least close to 900Gb
```
Comment (wasade) [inline qiita_pet/support_files/doc/source/faq.rst]: This statement will be inaccurate soon. I recommend not listing a size here. 

I did a test against study 2136, and Chrome immediately saw the download size, does the size info just get cached or something?

Comment (antgonza) [reply qiita_pet/support_files/doc/source/faq.rst]: Got it! I thought you meant that we should display beside the button the size, which is the difficult part; if having it in Chrome is sufficient, that's great! Note that the way it works is that python will create the list of files; nginx will take that info, create the zip (and the size reported in Chrome); and Chrome will do the rest. 

Anyway, what about replacing for: if you are unsure of the size required click on the button and your browser will show an estimate size of the download.

**Pull request merged**